tests/ami/packages_sanity_tests.py

```diff
@@ -229,21 +311,21 @@ def main(
             ex_security_groups=SECURITY_GROUPS,
             ssh_username=distro_details["ssh_username"],
             ssh_timeout=10,
-            timeout=140,
-            deploy=step,
+            timeout=240,
```

Comment:
Perhaps we should use different (longer) timeout just for Windows? Or you think there is a need to use it for Linux as well?

I spun up many Linux tests and it never needed longer to finish (unless there was some upstream issue which would cause it to time out in any case).

Reply:
I guess there were some timeouts and it was with Linux instances IIRC.  Job time should increase because now we upload agent package to install, so it can be quite slow.

Reply:
Also, It still does not have to wait for 240 seconds if it finishes earlier.